src/main/java/uk/gov/companieshouse/confirmationstatementapi/controller/NextMadeUpToDateController.java

```diff
@@ -1,35 +1,41 @@
 package uk.gov.companieshouse.confirmationstatementapi.controller;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RestController;
 import uk.gov.companieshouse.confirmationstatementapi.exception.CompanyNotFoundException;
 import uk.gov.companieshouse.confirmationstatementapi.model.json.NextMadeUpToDateJson;
 import uk.gov.companieshouse.confirmationstatementapi.service.ConfirmationStatementService;
 
+import java.util.HashMap;
+
+import static uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication.LOGGER;
+import static uk.gov.companieshouse.confirmationstatementapi.utils.Constants.ERIC_REQUEST_ID_KEY;
+
 @RestController
 public class NextMadeUpToDateController {
 
     @Autowired
     private ConfirmationStatementService confirmationStatementService;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(NextMadeUpToDateController.class);
-
     @GetMapping("/confirmation-statement/company/{companyNumber}/next-made-up-to-date")
-    public ResponseEntity<NextMadeUpToDateJson> getNextMadeUpToDate(@PathVariable String companyNumber) {
+    public ResponseEntity<NextMadeUpToDateJson> getNextMadeUpToDate(@PathVariable String companyNumber,
+            @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) {
```

Comment:
the param indentation looks a bit wonky, could you line them up like you have done in the FilingCcontroller